### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/blog/intro.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#750] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/blog/intro.html

主题通过引入 vuepress-plugin-blog2 提供了博客支持，该功能是默认禁用的。 如果你需要博客功能，你可以在主题选项中设置 plugins.blog: true 来启用博客功能。

https://theme-hope.vuejs.press/zh/guide/blog/intro.html

**Comment by Lil-Kr** [comment]
在使用markdown编辑文章之后, 文章的末尾会有一个 
上次编辑于: xxxx
贡献者: xxxx,

这个 “上次编辑于: ” 的时间是自动改的么？
